### PR TITLE
Add test for CVE-2024-27982

### PR DESCRIPTION
**Pull request by sergio-correia**

### CODE REVIEW

File: Security/cve-2024-27982-http-request-smuggling/main.fmf [hunk 1]
```diff
@@ -0,0 +1,14 @@
+summary: CVE-2024-27982 test
+description: Test for CVE-2024-27982
+contact: Patrik Koncity <pkoncity@redhat.com>
+test: ./runtest.sh
+recommend:
+- llhttp
+- rpm-build
+- dnf-plugins-core
+tag:
+  - CI-Tier-1
```
Comment (Koncpa) [inline Security/cve-2024-27982-http-request-smuggling/main.fmf]: I would remove Tier-1 tags, It won't be need to run it through every gating. WDYT?

File: Security/cve-2024-27982-http-request-smuggling/main.fmf [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  - Tier1
+tier: '1'
```
Comment (Koncpa) [inline Security/cve-2024-27982-http-request-smuggling/main.fmf]: Same as above, remove tier flag.

**Review by Koncpa — COMMENTED** [review]
Otherwise LGTM!

File: Security/cve-2024-27982-http-request-smuggling/main.fmf (hunk 1 above)
Comment (sergio-correia) [reply Security/cve-2024-27982-http-request-smuggling/main.fmf]: Removed.

File: Security/cve-2024-27982-http-request-smuggling/main.fmf (hunk 2 above)
Comment (sergio-correia) [reply Security/cve-2024-27982-http-request-smuggling/main.fmf]: Removed this as well.

**Pull request merged**